### PR TITLE
Only use serial crsf battery sensor if detected

### PR DESCRIPTION
**Pull request by JyeSmith**
If a receiver contains an internal battery sensor, and a user also connects a FC with vbat tlm, then they will both be sent to the handset.  This may cause confusion with the numbers being displayed, or alarms configured.

This PR disables devAnalogVbat (the internal battery sensor) if a serial crsf battery sensor is detected.

### CODE REVIEW

**Review by pkendall64 — APPROVED** [review]

File: src/lib/Telemetry/telemetry.h [hunk 1]
```diff
@@ -74,5 +76,6 @@ class Telemetry
     bool callEnterBind;
     bool callUpdateModelMatch;
     bool sendDeviceFrame;
+    bool crsfBatterySensorDetected;
```
Comment (wvarty) [inline src/lib/Telemetry/telemetry.h:79]: I could be wrong, but its probably not the best practice to assume this bool will initialise to false...

Comment (pkendall64) [reply src/lib/Telemetry/telemetry.h:79]: C++ guarantees that variables are created with a specific initial value. And for booleans that is the `false` value 😄

Comment (wvarty) [reply src/lib/Telemetry/telemetry.h:79]: Thats not what 5mins on stackoverflow taught me, but fair enough. I was always told that member vars should always be initialised (i.e. in the class constructor etc.)

Comment (wvarty) [reply src/lib/Telemetry/telemetry.h:79]: https://stackoverflow.com/questions/4622225/arent-boolean-variables-always-false-by-default

Comment (JyeSmith) [reply src/lib/Telemetry/telemetry.h:79]: Im happy to change it just for the readability.  But there are multiple instances that should be updated and probably not as part of this PR.

Comment (pkendall64) [reply src/lib/Telemetry/telemetry.h:79]: Yep, you are right Wez, only globals are automatically set to defaults.

**Review by wvarty — APPROVED** [review]
Approving this on the assumption that the un-initialised bool is working fine, given there are plenty of other examples in the code where it seems to be.

**Pull request merged**